README.md

```diff
@@ -69,15 +46,11 @@ See `ghcup --help`.
 * use [shellcheck](https://github.com/koalaman/shellcheck) and `checkbashisms.pl` from [debian devscripts](http://http.debian.net/debian/pool/main/d/devscripts/devscripts_2.18.4.tar.xz)
 * whitespaces, no tabs
 
-## TODO
+## Known problems
 
-- [ ] FreeBSD support ([#4](https://github.com/haskell/ghcup/issues/4))
-- [ ] Make fetching tarballs more robust ([#5](https://github.com/haskell/ghcup/issues/5))
-- [x] More code documentation
-- [x] Allow to compile from source ([#2](https://github.com/haskell/ghcup/issues/2))
-- [x] Allow to install cabal-install as well ([#3](https://github.com/haskell/ghcup/issues/3))
+### Limited distributions supported
 
-## Known problems
+Currently only GNU/Linux distributions compatible with the [upstream GHC](https://www.haskell.org/ghc/download_ghc_8_6_1.html#binaries) binaries (built on Fedora) are supported.
```

Comment:
Not accurate. We don't download the fedora builts unconditionally.

Reply:
fixed